src/pages/contact/index.tsx

```diff
@@ -0,0 +1,298 @@
+    },
+  },
+
+  navbar: {
+    marginBottom: theme.spacing(2.5),
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(3),
+    },
+    [theme.breakpoints.up('lg')]: {
+      marginBottom: theme.spacing(7),
+    },
+  },
+
+  inputs: {
+    display: 'grid',
+    gridTemplateRows: '1fr',
+    gridRowGap: theme.spacing(4),
+    [theme.breakpoints.up('sm')]: {
+      gridRowGap: theme.spacing(4),
+    },
+    [theme.breakpoints.up('lg')]: {
+      gridTemplateRows: '1fr',
+      gridGap: theme.spacing(4),
+    },
+  },
+
+  nameAndEmail: {
+    display: 'grid',
+    gridGap: theme.spacing(3),
+    '&.MuiFormControl-fullWidth': {
+      margin: theme.spacing(0, 2, 4, 2),
+    },
+    [theme.breakpoints.up('sm')]: {
+      justifyContent: 'space-between',
+      gridTemplateColumns: '1fr 1fr',
+      gridGap: theme.spacing(4),
+    },
+    [theme.breakpoints.up('lg')]: {
+      gridTemplateColumns: '1fr 1fr',
+      gridGap: theme.spacing(5),
+      justifyContent: 'space-between',
+    },
+  },
+
+  buttonWrapper: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'flex-end',
+    paddingTop: theme.spacing(4),
+  },
+
+  button: {
+    width: 158,
+    height: 40,
+    borderRadius: 8,
+    boxShadow: '0 2px 4px 0 rgba(47, 84, 235, 0.15), 0 8px 16px 0 rgba(47, 84, 235, 0.15)',
+  },
+
+  title: {
+    marginBottom: theme.spacing(4),
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(3),
+    },
+    [theme.breakpoints.up('lg')]: {
+      margin: theme.spacing(0, 2, 4, 2),
+    },
+  },
+}));
+
+const ContactPage = () => {
+  const classes = useStyles();
+
+  const developerProfile = useDeveloperProfile();
+  const { componentType, isDesktop, isMobile, isTablet } = useComponentType();
+
+  return (
+    <Container className={classes.container} maxWidth="lg">
+      <Helmet>
+        <title>Contact page</title>
+      </Helmet>
+      {isDesktop && (
+        <Box className={classes.detailsCard}>
+          <DetailsCard
+            fullName={`${developerProfile.firstName} ${developerProfile.lastName}`}
+            address={`${developerProfile.city}, ${developerProfile.country}`}
+            image={developerProfile.avatar.publicURL}
+            position={developerProfile.position}
+            socialMedia={[
+              { name: 'facebook', link: developerProfile.socialMedia.facebook },
+              { name: 'github', link: developerProfile.socialMedia.github },
+              { name: 'twitter', link: developerProfile.socialMedia.twitter },
+              { name: 'instagram', link: developerProfile.socialMedia.instagram },
+            ]}
+            phone={developerProfile.phone}
+            email={developerProfile.email}
+            isFreelancer={developerProfile.isFreelancer}
+            resumeLink={developerProfile.cv}
+            type={componentType}
+          />
+        </Box>
+      )}
+
+      <Box className={classes.main}>
+        {isDesktop && (
+          <Navbar
```

Comment:
Navbar should be defined as a separate component with all its props

Reply:
I created new issues